model/host/host.go

```diff
@@ -326,6 +326,8 @@ const (
 	DefaultUnexpirableHostsPerUser = 1
 	// Max total EBS volume size for user
 	DefaultMaxVolumeSizePerUser = 200
+	// Max number of volumes with no expiration for user
+	DefaultUnexpirableVolumesPerUser = 1
```

Comment:
should these constants be in globals.go, if not could they be private?

Reply:
done